### PR TITLE
Reorganized for WDI8

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -40,17 +31,17 @@ he or she likes, and on the right, three things he or she dislikes.
 
 Switch!
 
-For the remaining 10 minutes, we’ll go around the room and share!
+For the remaining 10 minutes, we'll go around the room and share!
```
Comment (jshawl) [inline readme.md:34]: technically should be ’ opt-shift-]

File: readme.md [hunk 2]
```diff
@@ -40,17 +31,17 @@ he or she likes, and on the right, three things he or she dislikes.
...
 
-Here’s a wireframe of Adrian as an example:
+Here's a wireframe of Adrian as an example:
```
Comment (jshawl) [inline readme.md:36]: same

File: readme.md [hunk 3]
```diff
@@ -99,7 +90,7 @@ Here’s a wireframe of Adrian as an example:
```
Comment (jshawl) [inline readme.md:90]: above line merged before review just fyi

File: readme.md [hunk 4]
```diff
@@ -246,73 +238,50 @@ If you complete 80% of the homework assigned over the duration of the class, and
 - The instructors and staff will be happy to write job recommendations for you
 - The Outcomes team will prioritize helping you in your job search
 
-We expect each of you to meet to these standards.
+If you fall below these critera, then you won't be considered "recommended for hire", and won't get these incentives. You can still attend class and attend most alumni events, and we'll still treat you like any other student and provide you with help, but we won't be comfortable putting that "GA Seal of Approval" on you. This is in order to protect our brand, and to benefit those who do meet the standards. Our philosophy is that being a good programmer is much more than writing good code.
```
Comment (jshawl) [inline readme.md:241]: can we change to positive instead of wont wont wont

File: readme.md [hunk 5]
```diff
@@ -246,73 +238,50 @@ If you complete 80% of the homework assigned over the duration of the class, and
...
 
-Our philosophy is that being a good programmer is much more than writing good code. There's no "bell curve" or number of students we have to give a "C" to. We're just asking you to be on-time and make an effort.
+### Homework
 
-### Attendance
+There will be roughly 2 hours of homework every night. You'll be submitting the homework through Github, which we'll cover later on this week. All assignments will be due on Thursday at 9am. Your squad instructor will provide meaningful feedback on one of these assignments and also on your lab. Robin will check all the other homework and provide feedback and answer questions if you ask.
 
-You'll be taking your own attendance every morning. We have a system set up on Garnet that tracks the IP address from which you take attendance -- so we can tell if you're not actually in a classroom!
+It's **really, really** important that you submit homework even if -- especially if -- you didn't complete it. We would much rather you spend 2 hours taking a stab at something and turn it in unfinished than you spend 10 hours and turn it in perfect.
```
Comment (jshawl) [inline readme.md:249]: or spend 10 hours on it and turn in nothing at all

File: readme.md [hunk 6]
```diff
@@ -246,73 +238,50 @@ If you complete 80% of the homework assigned over the duration of the class, and
...
+### Attendance
 
-- Additional Instructor Support is available during Evening Office Hours from 6:00 - 8:00PM on Tuesday and Wednesday Nights
-- Instructors who are not scheduled for Evening Office Hours **may not** respond to code questions via Slack or in person, even if we're in the office.
-  - **Question**: Why would we do this?
-  - **Answer**: Work/Life Balance.
-- Feel free to send any messages to us and we'll reach out to you the next day.
-- Should you have a logistical question, you may contact Adrian Maseda
-  - He is the **Student Communication Manager** for your cohort
+You'll be taking your own attendance every morning. We have a system set up on Garnet that tracks the IP address from which you take attendance -- so we can tell if you're not actually in a classroom!
```
Comment (jshawl) [inline readme.md:260]: feels a little big brothery - maybe make a joke about monitoring IP addresses without explicityl saying we are tracking you

File: readme.md (hunk 1 above)
Comment (RobertAKARobin) [inline readme.md:34]: I replaced them with standard quotes because it was rendering all weird on my computer.

**Pull request merged**